libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -35,6 +37,18 @@ export class PayPalManager {
     await this.client.baUpdate({ billingAgreementId, cancel: true });
   }
 
+  /**
+   * Create billing agreement using the ExpressCheckout token.
+   *
+   * If the call to PayPal fails, a PayPalClientError will be thrown.
+   *
+   */
+  async createBillingAgreement(token: string): Promise<string> {
+    const options: CreateBillingAgreementOptions = { token };
```

Comment:
You're effectively typecasting the object to this type by doing this style of definition.

It would be safer to do:

```
    const response = await this.client.createBillingAgreement({
      token,
    });
```

Reply:
Updated!